backend/app/services/knowledge/summary_service.py

```
@@ -0,0 +1,572 @@
+            db: Database session
+            kb_id: Knowledge base ID
+        """
+        if not settings.SUMMARY_ENABLED:
+            return
+
+        kb = (
+            db.query(Kind)
+            .filter(
+                Kind.id == kb_id,
+                Kind.kind == "KnowledgeBase",
+                Kind.is_active == True,
```

Ruff (0.14.10)

Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)